main/zoe/api/zoe-contract-facet.md

```diff
@@ -105,7 +103,7 @@ const inviteIssuer = await E(zoe).getInviteIssuer();
 - `keywords` `{Array <String>}`
 - Returns: <router-link to="/zoe/api/records.html#amountmathkeywordrecord">`{AmountMathKeywordRecord}`</router-link>
 
-Pass in an array of keywords and get an amountMathKeywordRecord in return.
+Pass in an array of keywords and get an `amountMathKeywordRecord` in return.
```

Comment:
This method doesn't exist anymore. We have `zcf.getAmountMath` which takes a brand argument and gives a single amountMath